tutorials/algorithms/01_algorithms_introduction.ipynb

```diff
@@ -89,7 +88,9 @@
    "source": [
     "## How to run an algorithm?\n",
     "\n",
-    "In order to run an algorithm we need to have backend, a simulator or real device, on which the circuits that comprise the algorithm can be run. So for example we can use the `aer_simulator_statevector` from the Aer provider."
+    "Algorithms rely on the primitives to evaluate expectation values or sample circuits. The primitives can be based on  a simulator or real device and can be used interchangeably in the algorithms, as they all implement the same interface.\n",
```

Comment:
```suggestion
    "Algorithms rely on the primitives to evaluate expectation values or sample circuits. The primitives can be based on a simulator or real device and can be used interchangeably in the algorithms, as they all implement the same interface.\n",
```

Reply:
Thanks, updated in [09051a4](https://github.com/Qiskit/qiskit-tutorials/pull/1370/commits/09051a41886769c541bbd227c62764cc63925ea4)!